doc/api/concatenated_openapi_spec.yaml

```diff
@@ -238,7 +250,25 @@ components:
                     $ref: '#/components/schemas/DataRepository'
                 connections:
                     $ref: '#/components/schemas/Connection'
-                timestamp:
+                experiment_id:
+                    type: string
+                user_id:
+                    type: string
+                status:
+                    type: string
+                    enum:
+                        - RUNNING
+                        - SCHEDULED
+                        - FINISHED
+                        - FAILED
+                        - KILLED
+                start_time:
+                    type: integer
+                end_time:
+                    type: integer
+                artifact_uri:
```

Comment:
A trained or saved model is an artifact of a run. We should have a possibility to retrieve the saved model from the run directly, so we should add `savedModelRepository` entry here. 
In all generality, a run might have multiple artifacts of any kind, as well as multiple saved models

Reply:
To my understanding, one Run can result in only one model, because it is the result of exactly one model definition with one Dataset and one infrastructure it runs on. 
Am I mistaken?
And if so, would you add an array of SavedModels per Run?

Reply:
Mlflow allows for multiple models per run. The ambiguity comes from the fact that we now added a model definition schema.
You are correct in that a saved model is determined by dataset, infrastructure and model definition (maybe also experiment).
Mlflow just doesn't know about the model definition.

We can either internally have a different run definition than mlflow and have a one-to-many mapping mlflowRun->mantikRun or we would need to drop the model definition in the run.

This is a design decision where @markusabel should be involved

Reply:
How about I'd extend the Run definition by a SavedModels field for now, and we discuss the details in the next refinement? So we can finally merge this one and #44.

Reply:
Please do so